variables.tf

```diff
@@ -383,3 +383,30 @@ variable "cpu_utilization_low_statistic" {
   default     = "Average"
   description = "The statistic to apply to the alarm's associated metric. Either of the following is supported: `SampleCount`, `Average`, `Sum`, `Minimum`, `Maximum`"
 }
+
+variable "default_alarms" {
+  type        = bool
+  default     = true
+  description = "Enable or disable cpu and memory Cloudwatch alarms"
+}
+variable "custom_alarms" {
```

Comment:
put a new line above

Reply:
I have no clue why this file is not getting updated